src/pages/settings/company/create/CompanyCreate.tsx

```diff
@@ -56,6 +60,16 @@ export function CompanyCreate(props: Props) {
 
     queryClient.invalidateQueries();
 
+    if (id) {
+      const basePage =
+        '/' +
+        (location.pathname.includes('/settings/gateways')
+          ? 'settings/online_payments'
+          : location.pathname.split('/')[1]);
+
+      navigate(basePage);
+    }
```

Comment:
Is it possible to refactor this using URL API?

Reply:
> Is it possible to refactor this using URL API?

@beganovich Hmm, I don't think so. We still need to split the pathname by '/', because even when using the URL API, we can't get exactly '/clients'. Screenshot:

![Screenshot 2024-04-15 at 17 56 39](https://github.com/invoiceninja/ui/assets/51542191/d3f47974-9959-4017-bbac-8d44e65de2c9)

Let me know your thoughts.